pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -43,6 +43,20 @@ self: super: {
     hinotify = if pkgs.stdenv.isLinux then self.hinotify else self.fsnotify;
   };
 
+  fsnotify = overrideCabal (super.fsnotify.override {
+    hinotify = if pkgs.stdenv.isLinux then self.hinotify else self.hfsevents;
+  }) (drv: {
+    doCheck = !pkgs.stdenv.isDarwin;
+  });
+
+  system-fileio = overrideCabal super.system-fileio (drv: {
+    doCheck = !pkgs.stdenv.isDarwin;
+  });
+
+  c2hs = overrideCabal super.c2hs (drv: {
+    doCheck = !pkgs.stdenv.isDarwin;
```

Comment:
Have you reported that test suite failure upstream? Is there maybe a ticket / issue URL that we could attach here in a comment?